### PR TITLE
Add trailing slash to URLs

### PR DESCRIPTION
**Pull request by ltagliaferri**
Resolving #43 by forcing a trailing slash to URLs and updating internal links throughout

Also resolving #39 

Signed-off-by: ltagliaferri

### CODE REVIEW

File: content/en/cosign/registry_support.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Today, Cosign has been tested and works against the following registries:
 * Sonatype Nexus Container Registry
 * Alibaba Cloud Container Registry
 
-We aim for wide registry support. To sign images in registries which do not yet fully support [OCI media types](https://github.com/sigstore/cosign/blob/main/SPEC.md#object-types), one may need to use `COSIGN_DOCKER_MEDIA_TYPES` to fall back to legacy equivalents. For example:
+We aim for wide registry support. To sign images in registries which do not yet fully support OCI media types, one may need to use `COSIGN_DOCKER_MEDIA_TYPES` to fall back to legacy equivalents. For example:
```
Comment (ltagliaferri) [inline content/en/cosign/registry_support.md:25]: This link and SPEC.md no longer exist (https://github.com/sigstore/cosign/blob/main/SPEC.md#object-types)

File: content/en/fulcio/oidc-in-fulcio.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The token must include the following claims:
 
 `job_workflow_ref` is included as a SAN URI: `https://github.com/{job_workflow_ref}`
 
-All other required claims are extracted and included in custom OID fields, as documented in [OID Information](oid-info.md).
+All other required claims are extracted and included in custom OID fields, as documented in [OID Information](https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md).
```
Comment (ltagliaferri) [inline content/en/fulcio/oidc-in-fulcio.md:100]: I believe this is meant to point to the repo docs

File: content/en/gitsign/FAQ.md [hunk 3]
```diff
@@ -47,28 +47,28 @@ The commit itself contains a signed digest of the user commit content (that is,
 the author, committer, message, etc.) along with the code signing certificate.
 This data is stored within the commit itself as part of your repository. Review
 guidance on
-[inspecting the Git commit signature](#inspecting-the-git-commit-signature) for
+[inspecting the Git commit signature](/gitsign/inspecting/) for
```
Comment (ltagliaferri) [inline content/en/gitsign/FAQ.md:50]: The Gitsign pages were restructured

File: nuxt.config.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
 import theme from '@nuxt/content-theme-docs'
 
 export default theme({
+  router: {
+    trailingSlash: true,
+  },
```
Comment (ltagliaferri) [inline nuxt.config.js:6]: This change appears to make the Netlify deploy preview work, we may also add the following to the config:

```
   sitemap: {
     hostname: 'https://docs.sigstore.dev',
     trailingSlash: true,
   },
```

**Review by ltagliaferri — COMMENTED** [review]
I deleted the **Troubleshooting** page because it came with this error message. I am not sure the original intention of the page, but we can always bring it back later.
Screenshot 2022-12-20 at 4 08 52 PM

**Review by vaikas — APPROVED** [review]

**Review by cpanato — APPROVED** [review]

**Pull request merged**